wagn/lib/decko/engine.rb

```diff
@@ -26,8 +26,8 @@ class Engine < ::Rails::Engine
       ActiveSupport.on_load(:active_record) do
         if defined? Wagn
           #this code should all be in Wagn somewhere, I suspect.
-          Wagn::Engine.paths['request_log'] = Wagn.paths['request_log']
-          Wagn::Engine.paths['log']         = Wagn.paths['log']
+          Engine.paths['request_log'] = Wagn.paths['request_log']
+          Engine.paths['log']         = Wagn.paths['log']
         else
           Cardio.card_config ::Rails.application.config
         end
```

Comment:
another way to think about this:

engine apps will obviously need to require Decko::Engine and NOT Decko::Application.  This should give them access to the controller, routes, and of course, the world of cards.  What else do they need?

It seems like the main other thing is the world of rake tasks (and eventually the CLI wrappers around them, which we think will use a separate "decko-rails" method.  some more comments on that below.

In any case, I'm worried that we're losing a lot of the meat here, because so many of these good discussions are in PR's.  I would suggest we add two new places for bigger issues:
1. a ticket on Wagn.org
2. a README on the decko-rails gem.  That would be a great place to hash out some API stuff. 

In general, I hope that README will be the main document for learning how to add a decko engine to your rails app, so it's a good place to write up the interface!